### DISCUSSION TITLE
Let's try out quarto together!

### DISCUSSION THREAD

**Discussion by xldrkp** [singlesourcepub/community#47] [Tools]
[quarto](https://quarto.org/) is *the* thing, I heard some people say. It merges many tools under the hood that you had to chain together yourself before. From a single source, various file types can be generated as well as articles, books and websites.

Some in our community think it's a good idea to try out quarto's features. So, let's do this together!

Feel free to post your experiences and challenges below as well as the documents you generate.

**Comment by mrchristian** [comment]
I'll fire it up, I need to do some tests with 3D model embedding into text and Wikidatab LoD content encoding :-) 

I need to render up a CbDD https://www.deckenmalerei.eu/ example wiki page section and embed some digital objects. Corpus der barocken Deckenmalerei in Deutschland (CbDD) is a heritage database of 'Corpus of Baroque ceiling frescos in Germany' from the Bavian Academy of Sciences https://deckenmalerei.badw.de/das-projekt.html

The first test would be a dining room (Die Tafelstube) as there are existing digital objects for this room and there are written descriptions https://www.deckenmalerei.eu/42d06165-58e7-4653-bfe4-3d5f7091fc33#7fb9a718-7e18-4053-81d9-3689f3f65548 (link slighty broken as images have SSL cert problem - Sept 22).

objects -  https://cloud.nfdi4culture.de/s/KdkKtHkSzQeEmKN?path=%2FTUM%20draft%20data%2F3D%20Weikersheim%20Tafelstube%2FTafelstube%20kompakkt%20obj

wikidata - https://mvp.tibwiki.io/wiki/Item:Q64

plus see if I can embed a computational part to query the LoD and content display
